contrib/unity/README

```diff
@@ -0,0 +1,285 @@
+Overview
```

Comment:
This file would be easier to read if it was `README.md`.

Reply:
Agreed. Will do that after the likely code changes.

Reply:
This should also be in the Github issue description.